Jenkinsfile.deploy

```diff
@@ -84,27 +86,29 @@ pipeline {
                         setDeployPropertyIfNeeded('git.author', getGitAuthor())
                         setDeployPropertyIfNeeded('project.version', getProjectVersion())
                         setDeployPropertyIfNeeded('release', isRelease())
-
+                        setDeployPropertyIfNeeded('optaplanner.version', getOptaPlannerVersion())
```

Comment:
Hi @radtriste , just for double checking. I can see an  'optaplanner.version' here and lines below an 'version.org.optaplanner' . Looks like one is for "deploy" and the second is for "maven", is this correct?

Reply:
yeah this one is just for the Jenkins job properties that we save at the end of the job run.